### DISCUSSION TITLE
[Auth] Trash feature 🗑️

### DISCUSSION THREAD

**Discussion by brog-io** [ente-io/ente#2182] [Enhancements]
It would probably be handy to have a trash system in Auth. Someone might accidentally delete something and lose access to their account.

**Comment by bazixv13** [comment]
seems to be the same as [#492](https://github.com/ente-io/ente/discussions/492)

**Reply by brog-io** [reply]
> seems to be the same as [#492](https://github.com/ente-io/ente/discussions/492)

You're right!
